src/SprykerEco/Yves/Payolution/Form/InvoiceSubForm.php

```diff
@@ -17,7 +17,7 @@
 
 class InvoiceSubForm extends AbstractPayolutionSubForm
 {
-    const PAYMENT_METHOD = 'invoice';
+    public const PAYMENT_METHOD = 'invoice';
```

Comment:
Please check if It might be protected. Also check const calling using `static` instead of `self`.